Doc/library/io.rst

```diff
@@ -1072,13 +1073,14 @@ Text I/O
 
       import io
 
-      output = io.StringIO()
-      output.write('First line.\n')
+      output = io.StringIO('Start text')
```

Comment:
Please put your demo in a second example and leave this one intact.  This one shows the normal and typical use pattern.

Reply:
Ok, sure.
Thank you for the review